flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -168,6 +170,11 @@ public Builder table(Table newTable) {
       return this;
     }
 
+    public Builder properties(Map<String, String> newProps) {
```

Comment:
let's add a Javadoc to explain the motivation/usage of this API

Reply:
May I suggest the following? I don't think we need to add trivial `@param` and `@return`

```
    /**
     * Set the write properties for Flink sink. Write properties (with the same keys) defined in table properties
     * will be overwritten by properties provided here.
     */
```

Reply:
thx. done.